package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "build": "ncc build src/index.js",
     "lint": "eslint .",
-    "test": "tap",
+    "test": "tap test/*.test.js",
```

Comment:
why this change?

Reply:
locally without it, the command tooks ~2/3 seconds more to start

Reply:
did you figure out why?